easybuild/easyconfigs/r/RDKit/RDKit-2019.09.1-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,95 @@
+easyblock = 'CMakeMake'
+
+name = 'RDKit'
+version = '2019.09.1'
+versionsuffix = '-Python-%(pyver)s'
+
+local_ringdecomposerlib_ver = '1.1.3_rdkit'
+
+homepage = 'https://www.rdkit.org'
+description = "RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python."
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [
+    'https://github.com/rdkit/rdkit/archive/',
+    'https://github.com/rareylab/RingDecomposerLib/archive/',
+    # see https://www.inchi-trust.org/downloads/
+    'https://www.inchi-trust.org/download/105/',
+    'https://sourceforge.net/projects/avalontoolkit/files/AvalonToolkit_1.2/',
+]
+sources = [
+    'Release_%s.tar.gz' % version.replace('.', '_'),
+    'v%s.tar.gz' % local_ringdecomposerlib_ver,
+    # add datestamp to unversioned INCHI-1-SRC.zip based on date of most recent modified file in zipball
+    {'download_filename': 'INCHI-1-SRC.zip', 'filename': 'INCHI-1-SRC-20170203.zip'},
+    {'filename': 'AvalonToolkit_1.2.0.source.tar', 'extract_cmd': "cp %s ."},
```

Comment:
Should probably have a local_avalon_ver here and reuse that in the URL specification